src/data/graphql/typeDefs.ts

```diff
@@ -209,6 +209,14 @@ export default gql`
     vote: Int!
   }
 
+  type MotionVoteResults {
+    currentUserVoteSide: Int!
+    yayVotes: String!
+    yayVoters: [String!]!
+    nayVotes: String!
+    nayVoters: [String]!
```

Comment:
Why is nayVoters `[String]!` but yayVoters `[String!]!`?

Reply:
Good catch!